NHSDigital/validated-relationships-service-api
# NPA-3887: Added tests to postman collection

## Pull Request Checklist

## Ticket Link

https://nhsd-jira.digital.nhs.uk/browse/NPA-3887

## Description/Change Summary

- Added tests to postman collection
- Added additional requests to cover more sandbox scenarios
- Renamed existing requests

# How to test?
1. Import this collection into Postman
2. Run all tests in the collection

There should be 2 tests that fail for the identifier system request - These are known issues to be fixed.
